code/modules/surgery/organs/helpers.dm

```diff
@@ -0,0 +1,35 @@
+/mob/proc/get_int_organ(typepath) //int stands for internal
+	return
+
+/mob/proc/get_organs_zone(zone)
+	return
+
+/mob/proc/get_organ_slot(slot) //is it a brain, is it a brain_tumor?
+	return
+
+/mob/living/carbon/get_int_organ(typepath)
+	return (locate(typepath) in internal_organs)
```

Comment:
I'm not sure how nicely this will cope with a human having multiple of the same organ path